### PR TITLE
Another round of README creation and updates.

### PR DESCRIPTION
**Pull request by bobmcwhirter**
- [ ] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [ ] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [ ] Have you built the project locally prior to submission with `mvn clean install`?

-----

### CODE REVIEW

File: fractions/javaee/jaxrs-cdi/README.adoc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# JAX-RS + CDI
+
+An internal fraction providing integration between JAX-RS
+and CDI.
+
+Please see the documentation for the JAX-RS and CDI fractions.
```
Comment (tradej) [inline fractions/javaee/jaxrs-cdi/README.adoc]: The convention across the docs organisation is to write these sentences like this:

_For more information, see the JAX-RS or CDI fraction documentation._

Here, _JAX-RS_ and _CDI_ are links to the docs in question, or:

_For more information, see the CHAPTER_NAME chapter of the JAX-RS documentation, or the CHAPTER_NAME chapter of the CDI fraction documentation._ 

Here, _CHAPTER_NAME_ are links to the specific chapters of the respective documentations.

You can also write the sentences without linking to the documentations, but it facilitates the reader's job a great deal to be able to just click a link; aside from that, it is considered a good practice.

**Review by tradej — CHANGES_REQUESTED** [review]

File: fractions/javaee/jaxrs-cdi/README.adoc (hunk 1 above)
Comment (bobmcwhirter) [reply fractions/javaee/jaxrs-cdi/README.adoc]: That makes sense.

Comment (bobmcwhirter) [reply fractions/javaee/jaxrs-cdi/README.adoc]: If acceptable, I'll avoid links for now, as these READMEs are 

* Useful (maybe?) from with GitHub, which has different structure than the ref-guide where they also get slurped, and have (at the moment) different link/directory structures.
* The ref-guide is being restructured (Real Soon Now) and no links seems better than possibly broken links.

**Review by tradej — APPROVED** [review]
I see you decided not to create the links. This is acceptable, even though I slightly frown upon it. :)

**Pull request merged**